### PR TITLE
[release] version 3.0.3

### PR DESCRIPTION
**Pull request by zhongjiajie**

### CODE REVIEW

**Comment by zhongjiajie** [comment]
I also change some things auto fix by pre-commit

File: docs/docs/en/contribute/release/release.md [hunk 1]
```diff
@@ -469,10 +469,10 @@ the release version is `<VERSION>`, the following updates are required(note it w
   - `docs/configs/site.js`:
     - `docsLatest`: update to `<VERSION>`
     - `docs0`: The `text` of two places of `en-us/zh-cn` needs to be updated to `latest(<VERSION>)`
-  - `docs/configs/index.md.jsx`: Add `<VERSION>: docsxyzConfig`
+  - `docs/configs/index.md.jsx`: Add `<VERSION>: docsxyzConfig` and add new `import` for the new `docsxyzConfig`
   - `docs/docs/en/history-versions.md` and `docs/docs/zh/history-versions.md`: Add new `<VERSION>` release docs.
   - `.github/ISSUE_TEMPLATE/bug-report.yml`: DolphinScheduler's GitHub [bug-report](https://github.com/apache/dolphinscheduler/blob/dev/.github/ISSUE_TEMPLATE/bug-report.yml)
-    issue template have **Version** selection bottom. So after we release DolphinScheduler we should and the new `<VERSION>` to
+    issue template have **Version** selection bottom. So after released we should and the new `<VERSION>` to
```
Comment (Tianqi-Dotes) [inline docs/docs/en/contribute/release/release.md:475]: we should and the new `<VERSION>`
maybe 
we should add the new `<VERSION>`

**Review by Tianqi-Dotes — APPROVED** [review]
lgtm

File: docs/docs/en/contribute/release/release.md (hunk 1 above)
Comment (zhongjiajie) [reply docs/docs/en/contribute/release/release.md:475]: will and in the next pr

**Pull request merged**